### PR TITLE
Fixed searchbar theme issue #74

### LINKED ISSUE #73: Searchbar
As per [Figma](https://www.figma.com/file/e6zTS3UndDm7IvwLXXSkOe/NFT-Dashboard-Light%2FDark-(Community)?node-id=13%3A991&t=PPftxhZ2UwSTwMaL-0)

### Preview

<div align="center">
 
 
</div>

### Variant 

- [x] Light
- [x] Dark

### PR DESCRIPTION
**Pull request by Kamasah-Dickson**
- Fixed duplicate import in the head component
 - Fixed logo supporting only dark themed color
 - Updated color scheme to support dark theme
 

Fixes #74

### CODE REVIEW

**Comment by Kamasah-Dickson** [comment]
## What do you think?
localhost_3000_ (1)
localhost_3000_ (2)

**Comment by mrxshinji** [comment]
It should link to #73 not #74

And it look different from the preview.

Also dont touch the current header, do it something like this PR
https://github.com/appwex/react-ui-components/pull/67

**Comment by abdulmaleekadams** [comment]
cool

**Review by abdulmaleekadams — COMMENTED** [review]
Looks good

**Comment by Kamasah-Dickson** [comment]
I never changed anything I just added the darkMode as I was told

**Comment by abdulmaleekadams** [comment]
@Kamasah-Dickson Yea I can see that did you use emotion css

**Comment by Kamasah-Dickson** [comment]
No I used just normal CSS

**Comment by mrxshinji** [comment]
Have you split the headbar into separate component?

**Comment by Kamasah-Dickson** [comment]
No review the changes. It is a component for the theme functionality.......I seperated the functionality from the header

File: src/components/header/header.css [hunk 1]
```diff
@@ -78,6 +83,13 @@
   background: rgba(242, 242, 247, 0.5);
 }
 
+.darkSearch {
+  background-color: rgba(22, 22, 34, 0.979);
+}
+.darkSearch input {
```
Comment (aftabrehan) [inline src/components/header/header.css:89]: Please, make sure to leave a blank line between every selector. Thank you!

```suggestion
}

.darkSearch input {
```

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @Kamasah-Dickson 💪 🚀 💯 

It just needs a minor fix, and then it's good to go 🚀

**Pull request merged**